hooks/pre-exit

```diff
@@ -23,7 +23,7 @@ fi
 ### Step 2: Run `Pkg.gc` to try to reduce the size of the depot.
 
 echo "Running Pkg.gc()"
-julia --color=yes -e 'using Pkg; using Dates; Pkg.gc()'
+julia --color=yes -e 'using Pkg; using Dates; Pkg.gc(; collect_delay=Day(1), verbose=true)'
```

Comment:
What about Julia 1.6? `collect_delay` isn't there. Can't we just use the default of 7 days?

Reply:
You can actually use `collect_delay`, it just isn't documented.

```julia
julia> VERSION
v"1.6.2"

julia> using Pkg, Dates

help?> Pkg.gc
  Pkg.gc(; io::IO=stderr)

  Garbage collect packages that are no longer reachable from any project. Only packages
  that are tracked by version are deleted, so no packages that might contain local
  changes are touched.

julia> Pkg.gc(; collect_delay=Day(1), verbose=true)
      Active manifest files: 1 found
        `~/.julia/environments/v1.8/Manifest.toml`
      Active artifact files: 0 found
      Active scratchspaces: 0 found
     Deleted no artifacts, repos, packages or scratchspaces
```

Reply:
That being said, I agree we need to tune that parameter.

So I've set `collect_delay=Day(7)`, which is equivalent to the default, but by listing it explicitly, it'll make it easier in the future to make PRs to modify that value.

Reply:
I see; good to know.
Is it still worth specifying if the default is 7 days?

Reply:
> I see; good to know.
> Is it still worth specifying if the default is 7 days?

I was thinking we might toy with increasing/decreasing it in the future, so if we leave it in there as "boilerplate", it's easier to just edit the numerical value.

I can also take it out for now, and we can add it back later?